lib/openqa.rb

```diff
@@ -138,9 +138,12 @@ def notify_mail
   def build_email_message
     message = {}
     message['to'] = 'obs-tests@opensuse.org'
-    emoji = '✔️'
+    emoji = '💬'
```

Comment:
I would say we should not notify *at all* for the other states

Reply:
I modified the PR. Please have a look.